honcho/command.py

```diff
@@ -33,9 +32,16 @@
     'procfile': 'PROCFILE',
 }
 
-export_choices = dict((_export.name, _export)
-                      for _export in iter_entry_points('honcho_exporters'))
+try:
+    from pkg_resources import iter_entry_points # Python 3.7-3.9
 
+    export_choices = dict((_export.name, _export)
+                          for _export in iter_entry_points('honcho_exporters'))
+except ImportError:
+    from importlib.metadata import entry_points # Python 3.10+
+
+    export_choices = dict((_export.name, _export)
+                          for _export in entry_points(group='honcho_exporters'))
```

Comment:
this should really be flipped -- prefer `importlib.metadata` and if it's not available then fall back to (deprecated) `pkg_resources` -- otherwise this will still trigger deprecation warnings even on systems where `importlib.metadata` would avoid such warnings